packages/backend/src/entity/controller.ts

```diff
@@ -0,0 +1,66 @@
+import { RouteConfig } from '@asteasolutions/zod-to-openapi'
+import { z } from 'zod'
+import { Entity, Schema } from '@ssi-trust-registry/common'
+import { Request, Response } from 'express'
+import partial from 'lodash.partial'
+import { EntityService } from './service'
+import { SchemaService } from '../schema/service'
+import { createLogger } from '../logger'
+import { RequestWithToken } from '../auth/middleware'
+
+const logger = createLogger(__filename)
+
+const RegistryResponseSchema = z.object({
+  entities: z.array(Entity),
+  schemas: z.array(Schema),
+})
+
+export function createRegistryController(
+  entityService: EntityService,
+  schemaService: SchemaService,
+): EntityController {
+  return {
+    getRouteConfigDocs,
+    getRegistry: partial(getRegistry, entityService, schemaService),
+  }
+}
+
+export interface EntityController {
```

Comment:
Why `EntityController`?

Reply:
You are right. It should be `RegistryController`